internal/contributebot/README.md

```diff
@@ -77,18 +77,24 @@ go run . --project=your-project-name --github_app=42 --github_key=/foo.pem
 
 ## Deploying
 
+To deploy an updated Contribute Bot to production, follow these steps.
```

Comment:
Since it is reasonable to want to stage an infrastructure change, also include some verbage on how to change the process to fit your dev environment?

Reply:
Done.